src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  } else {
+    if (days < 7 && days >= 3) {
+      return (days * 40) - 20;
```

Comment:
move to variables to everyone could understand where these numbers came from and what is it

Reply:
did as you said